.env.example

```diff
@@ -66,6 +66,11 @@ DB_SEEDS_DIR="./src/database/seeds"
 AUTH0_HOST="http://localhost:3333"
 
 #
+# RPC api authentication
+#
+USER_NAME="particl"
+USER_PASSWORD="particl_123"
+
```

Comment:
remove USER_NAME and USER_PASSWORD.
use RPCUSER and RPCPASSWORD as the login and password required for the rpc api, that way the marketplace api will use the same ones that are defined for the particld rpc api.